cogs/wordfilter/testWordFilter.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from .wordfilter import (
+    _filterWord,
+    _isAllFiltered,
+    _isOneWord,
+)
+import re
```

Comment:
Might as well organize these imports.
```suggestion
import re

import pytest

from .wordfilter import (
    _filterWord,
    _isAllFiltered,
    _isOneWord,
)
```
Your thoughts?